web/app/routes/post.$year.$date.$slug.tsx

```diff
@@ -196,6 +212,74 @@ export default function ArticleRoute() {
         </header>
         <Article post={data} />
       </div>
+      <div>
+        <InstantSearch searchClient={searchClient} indexName={algolia.index}>
+          <RelatedProducts
+            headerComponent={() => (
+              <div className="inset-0 flex m-6 justify-center ">
+                <DoorSign>Relaterte artikler</DoorSign>
+              </div>
+            )}
+            objectIDs={[data._id]}
+            limit={3}
+            layoutComponent={({ items }) => (
+              <RelatedPostsLayout
+                items={items.map(
+                  (item) =>
+                    ({
+                      objectID: item.objectID,
+                      name: item.title,
+                      image: item.image,
+                      author: item.authors,
+                      tags: item.tags || [],
+                      slug: item.slug.current,
+                      availableFrom: item.availableFrom,
+                    }) as RelatedPostsData
+                )}
+              />
+            )}
+            emptyComponent={() => <p className="text-black">Ingen anbefalinger</p>}
+          />
+        </InstantSearch>
+      </div>
+    </div>
+  )
+}
+
+interface RelatedPostsData {
+  objectID: string
+  name: string
+  author: string[]
+  tags: string[]
+  slug?: string
+  availableFrom?: string
+}
+
+const RelatedPostsLayout = ({ items }: { items: RelatedPostsData[] }) => {
+  return (
+    <div className="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-4 m-4 pb-10">
+      {items.map((item) => {
+        if (!item || !item.availableFrom) return null
+        const date = parseDate(item.availableFrom)
+        return (
+          <div key={item.objectID} className="striped-frame border p-4 rounded-lg shadow-md bg-postcard-beige">
+            <Link to={`/post/${date.year}/${date.day}/${item.slug}`} className={'flex flex-col justify-between'}>
```

Comment:
date.day burde alltid ha en 0 foran seg på datoer før den 10. 

```suggestion
            <Link to={`/post/${date.year}/${String(date.day).padStart(2, "0")}/${item.slug}`} className={'flex flex-col justify-between'}>
```